### PR TITLE
fix(webapp): correct percent

### LINKED ISSUE #257: Claimed % is wrong
In this problem we must take as maximum expense the amount that has been claimed

Image

### PR DESCRIPTION
**Pull request by JAMares**
### What does this PR do?

- Resolve #257

### Steps to test

1. Run project
2. Enter to Income and Expense, change to 'one election' and must something like:
imagen

#### CheckList

- [X] Follow proper Markdown format
- [X] The content is adequate
- [X] The content is available in both english and spanish
- [X] I Ran a spell check

### CODE REVIEW

File: hapi/src/services/dfuse/updaters/genesiseden-withdraw.updater.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ module.exports = {
 
       if (!registeredTransaction) await edenTransactionGql.save(transactionData)
     } catch (error) {
-      console.error(`error to sync ${action.action}: ${error.message}`)
+      console.error(`error to sync withdraw ${action.action}: ${error.message}`)
```
Comment (xavier506) [inline hapi/src/services/dfuse/updaters/genesiseden-withdraw.updater.js:28]: withdrawal sync error

File: hapi/src/services/dfuse/updaters/genesiseden-fundtransfer.updater.js [hunk 2]
```diff
@@ -34,7 +34,9 @@ module.exports = {
       try {
         await edenTransactionGql.save(transactionData)
       } catch (error) {
-        console.error(`error to sync ${action.action}: ${error.message}`)
+        console.error(
+          `error to sync fundtransfer ${action.action}: ${error.message}`
```
Comment (xavier506) [inline hapi/src/services/dfuse/updaters/genesiseden-fundtransfer.updater.js:38]: fundtransfer sync error

File: hapi/src/services/dfuse/updaters/eosiotoken-transfer.updater.js [hunk 3]
```diff
@@ -39,7 +39,7 @@ module.exports = {
 
       if (!registeredTransaction) await edenTransactionGql.save(transactionData)
     } catch (error) {
-      console.error(`error to sync ${action.action}: ${error.message}`)
+      console.error(`error to sync transfer ${action.action}: ${error.message}`)
```
Comment (xavier506) [inline hapi/src/services/dfuse/updaters/eosiotoken-transfer.updater.js:42]: transfer sync error

File: webapp/src/routes/ExpenseReport/index.js [hunk 4]
```diff
@@ -65,6 +65,8 @@ const ExpenseReport = () => {
           ...firstObj
         }))
 
+  console.log(tableData)
```
Comment (xavier506) [inline webapp/src/routes/ExpenseReport/index.js:68]: do we need these?

File: webapp/src/routes/IncomeReport/index.js [hunk 5]
```diff
@@ -66,6 +66,8 @@ const IncomeReport = () => {
           ...firstObj
         }))
 
+  console.log(tableData)
```
Comment (xavier506) [inline webapp/src/routes/IncomeReport/index.js:69]: do we need this?

File: webapp/src/routes/ExpenseReport/index.js (hunk 4 above)
Comment (JAMares) [reply webapp/src/routes/ExpenseReport/index.js:68]: i miss deleted it

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**